src/Schema/AbstractSchemaManager.php

```diff
@@ -1735,12 +1735,17 @@ public function getSchemaSearchPaths()
      *
      * @param string|null $comment
      * @param string      $currentType
+     * @psalm-param non-empty-string|null $comment
```

Comment:
Do we need this? A `''` as comment wouldn't do any harm as far as I can tell. The regex simply wouldn't match which is okay.